diff -c flexdll-0.34.orig/reloc.ml flexdll-0.34.patched/reloc.ml
*** flexdll-0.34.orig/reloc.ml	2015-01-22 17:30:07.000000000 +0100
--- flexdll-0.34.patched/reloc.ml	2015-10-27 09:01:06.146099400 +0100
***************
*** 346,351 ****
--- 346,354 ----
      ) else true
    in
    let section sec =
+     (* M.SOEGTROP: disabled cause of problems with lablgtk for mingw64 build on cygwin.
+        See https://github.com/alainfrisch/flexdll/commit/37e6b5ad904b0d4648cebb09c19ed10e6f8dea28
+        
      if sec.sec_opts &&& 0x1000l <> 0l && has_prefix ".rdata$.refptr." sec.sec_name then
        begin
          (* under Cygwin64, gcc introduces mergable (link once) COMDAT sections to store
***************
*** 354,359 ****
--- 357,363 ----
          sec.sec_opts <- 0xc0500040l;
          sec.sec_name <- Printf.sprintf ".flexrefptrsection%i" (Oo.id (object end));
        end;
+     *)
  
      let min = ref Int32.max_int and max = ref Int32.min_int in
      let sym = lazy (let s = Symbol.intern sec 0l in
Common subdirectories: flexdll-0.34.orig/test and flexdll-0.34.patched/test
